resip/stack/ssl/TlsConnection.cxx

```diff
@@ -206,6 +206,11 @@ TlsConnection::checkState()
       else
       {
          InfoLog( << "TLS handshake starting (client mode)" );
+         /* OpenSSL version < 0.9.8f does not support SSL_set_tlsext_host_name() */
+         #if defined(SSL_set_tlsext_host_name)
```

Comment:
please move the macro lines (#if and #endif) all the way to the left, no indentation

move the code 3 spaces to the left, so it is aligned with the rest of the code in this block